### DISCUSSION TITLE
Updating frequency slot after changing default channel

### DISCUSSION THREAD

**Discussion by stanhamil** [meshtastic/Discussions#154] [General]
Can someone please confirm my understanding of something?

My primary channel (0) is used for a private channel and the first, secondary channel (1) is used for the default, public channel.

Being on the EU 868mhz band, am I correct in thinking I need to manually change the frequency slot my device operates on to “1”, as this is usually what would be set if I had “LongFast” set up as my primary channel name?

**Comment by ianmcorvidae** [comment]
In most regions, you do need to set the frequency slot, but EU_868 is special, in a sense, because it only has one frequency slot to use (with the LONG_FAST modem preset). You're right that slot 1 is the one you need to use, but also it's the only available slot, so you don't really need to change anything in order to end up on the correct slot.

**Reply by stanhamil** [reply]
In the LoRa settings, I found that the frequency slot was “0”, so I changed this to “1”.

I believe it reverted to “0” because I changed the primary channel from the default name. I found the following information [here](https://meshtastic.org/docs/configuration/tips/):

> If your LoRa frequency slot is set to the default (0), the radio's transmit frequency will be automatically changed based on your PRIMARY channel's name. In this case, you will have to manually set it back to your region's default (in LoRa settings) in order to interface with users on the default slot.

This implies that the frequency slot isn’t necessarily correct if you change the name of the primary channel.
